runtime/java.go

```diff
@@ -52,14 +63,25 @@ func (conf javaRuntime) GetBootCmd(cmdConfs map[string]*CmdConfig, env map[strin
 	if conf.Base == "" { // Allow user to use e.g. "openjdk7:java" package instead default one.
 		conf.Base = "openjdk8-zulu-compact1:java"
 	}
-	conf.setDefaultEnv(map[string]string{
-		"XMS":       conf.Xms,
-		"XMX":       conf.Xmx,
-		"CLASSPATH": strings.Join(conf.Classpath, ":"),
-		"JVM_ARGS":  conf.concatJvmArgs(),
-		"MAIN":      conf.Main,
-		"ARGS":      strings.Join(conf.Args, " "),
-	})
+
+	// Only set java-specific environment variables when base is openjdk-like.
+	if isCompatibleBase(conf.Base, javaPackages) {
+		if len(conf.Classpath) == 0 {
+			conf.Classpath = append(conf.Classpath, "/")
```

Comment:
What does this do? `Classpath` is an array, and here you are adding root into the classpath. Is this because the java packages are "weirdly" prepared or because of a specific application package?

Reply:
The chosen value, yes :smile:. However, now that you've explained it, I understand that it's easier for the users, so I am no longer against it.